### ISSUE TITLE
Use of "extends" in Tile.ViewerRequestVolume schema

### ISSUE THREAD

**Issue by amenzies** [AnalyticalGraphicsInc/3d-tiles#273]
I'm by no means an expert in schema but I noticed that in [tile.schema.json](https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/schema/tile.schema.json#L13) `viewerRequestVolume` extends `boundingVolume.schema.json` rather than having a direct `$ref` like `boundingVolume` as seen here:
```
        "boundingVolume" : {
            "description" : "The bounding volume that encloses the tile.",
            "$ref" : "boundingVolume.schema.json"
        },
        "viewerRequestVolume" : {
            "extends" : { "$ref" : "boundingVolume.schema.json" },
            "description" : "Optional bounding volume that defines the volume that the viewer must be inside of before the tile's content will be requested and before the tile will be refined based on geometricError."
        },
```
I just wanted to check to see if this is intentional.  I noticed this when using [NJsonSchema](https://www.nuget.org/packages/NJsonSchema.CodeGeneration.CSharp/) to generate C# code directly from the schema.  The result is that the generated code uses the type `object` for `viewerRequestVolume` instead of `BoundingVolume`.  This may not be an issue with the schema and may just be a quirk of NJsonSchema.  I'm guessing there is probably a good reason to prefer the use of `extends` that I am unaware of, in which case please feel free to disregard this ticket.

**Comment by lilleyse** [comment]
I'm honestly not sure, but I don't see the harm in just using a `$ref`. It looks like `extends` was removed after version 3 of JSON Schema anyways.

**Comment by pjcozzi** [comment]
@amenzies thanks again for the report, this was fixed in #274.

Also, please let us know if you are building something cool with 3D Tiles - we would love to see it!
